app/test_main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+from unittest.mock import patch
+
+from app.main import can_access_google_page
+
+
+def test_cannot_access_if_only_valid_url() -> None:
```

Comment:
The test name `test_cannot_access_if_only_valid_url` does not accurately reflect what is being tested. A more descriptive name could be `test_cannot_access_if_only_url_is_valid` as it better reflects that the test checks the scenario where the URL is valid but there is no internet connection.